tests/atomic_swap/test_get_info.py

```diff
@@ -0,0 +1,209 @@
+"""
+Provide tests for command line interface's atomic swap commands.
+"""
+import json
+import re
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    ADDRESS_REGEXP,
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_27_IN_TESTNET_ADDRESS,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+    SWAP_IDENTIFIER_REGEXP,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+SWAP_IDENTIFIER_PRESENTED_ON_THE_TEST_NODE = '033402fe1346742486b15a3a9966eb5249271025fc7fb0b37ed3fdb4bcce6808'
+
+
+def test_get_swap_info():
+    """
+    Case: get information about atomic swap by its identifier.
+    Expect: information about swap is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'atomic-swap',
+        'get-info',
+        '--id',
+        SWAP_IDENTIFIER_PRESENTED_ON_THE_TEST_NODE,
+        '--node-url',
+        NODE_27_IN_TESTNET_ADDRESS,
+    ])
+
+    swap_info = json.loads(result.output).get('result').get('information')
+    swap_identifier = swap_info.get('swap_id')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert SWAP_IDENTIFIER_PRESENTED_ON_THE_TEST_NODE == swap_identifier
+    assert re.match(pattern=ADDRESS_REGEXP, string=swap_info.get('sender_address')) is not None
+    assert re.match(pattern=ADDRESS_REGEXP, string=swap_info.get('receiver_address')) is not None
+    assert re.match(pattern=SWAP_IDENTIFIER_REGEXP, string=swap_identifier) is not None
+    assert isinstance(int(float(swap_info.get('amount'))), int)
```

Comment:
What is this test here for? We will always have an int.

Reply:
@Alladin9393, done.